guid_helpers/src/guid_helpers.rs

```diff
@@ -0,0 +1,85 @@
+#![cfg_attr(target_os = "uefi", no_std)]
+
+use r_efi::efi;
+use uuid::uuid;
+
+/// Macro for creating an `efi::Guid` from string representation.
+/// This is a wrapper for `uuid!` from the uuid crate.
+#[macro_export]
+macro_rules! guid {
+    ($guid_str:expr) => {
+        efi::Guid::from_bytes(&uuid!($guid_str).to_bytes_le())
+    };
+}
+
+/// Macro for printing an `efi::Guid` as a string.
+#[macro_export]
+macro_rules! guid_fmt {
+    ($guid_object:expr) => {
+        format_args!("{:X}", uuid::Uuid::from_bytes_le(*$guid_object.as_bytes()))
+    };
+}
+
+/// Macro for creating a `uuid::Uuid` from an `efi::Guid`.
+#[macro_export]
+macro_rules! guid_to_uuid {
+    ($guid_object:expr) => {
+        uuid::Uuid::from_bytes_le(*$guid_object.as_bytes())
+    };
+}
+
+const ZERO_GUID_STR: &str = "00000000-0000-0000-0000-000000000000";
+
+pub const ZERO_GUID: efi::Guid = guid!(ZERO_GUID_STR);

```

Comment:
Do you think it makes sense to put these GUID definitions in a new module called something like `Guid` so the name space can be `Guid::CALLER_ID` and `Guid::ZERO`? That seems to leverage the Rust namespace a bit more to organize the content.

Reply:
Renamed module from `guid_helpers` to `guid`, renamed const GUIDs to match recommendation. Re-tested with unit tests and in hardware.